omnigibson/action_primitives/curobo.py

```diff
@@ -361,22 +465,37 @@ def compute_trajectories(
         """
         # Previously, this would silently fail so we explicitly check for out-of-range joint limits here
         # This may be fixed in a recent version of CuRobo? See https://github.com/NVlabs/curobo/discussions/288
-        if not th.all(th.abs(self.robot.get_joint_positions(normalized=True))[:-2] < 0.99):
-            print("Robot is near joint limits! No trajectory will be computed")
-            return None
-
-        # Make sure a valid (>1) number of entries were submitted
-        for tensor in (target_pos, target_quat):
-            assert (
-                len(tensor.shape) == 2 and tensor.shape[0] > 1
-            ), f"Expected inputted target tensors to have shape (N,3) or (N,4), where N>1! Got: {tensor.shape}"
+        # relevant_joint_positions_normalized = (
```

Comment:
is this needed?

Reply:
Not entirely sure. Seems not needed. I will leave them commented out for now in case we need them later.